src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,109 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final int DEFAULT_CAPACITY = 10;
+    private static final double INCREASE_MULTIPLIER = 1.5;
+    private int defaultInernalCapacity = 10;
```

Comment:
You don`t using defaultInernalCapacity in your code, and you already have DEFAULT_CAPACITY.